flink-runtime-web/web-dashboard/src/app/pages/job/overview/detail/job-overview-drawer-detail.component.ts

```diff
@@ -39,6 +40,12 @@ export class JobOverviewDrawerDetailComponent implements OnInit, OnDestroy {
   public ngOnInit(): void {
     this.jobService.selectedVertex$.pipe(takeUntil(this.destroy$)).subscribe(node => {
       this.node = node;
+      if (this.node != null && this.node.description != null) {
+        if (this.node.description.indexOf('<br/>') > 0) {
+          this.multilineNameCSS = 'name-multi-line';
+          this.node.description = this.node.description.replace(/<br\/>/g, '\n');
```

Comment:
Could you elaborate a bit why we need to replace `<br/>` back to `\n`?

Reply:
In my experiment, the html tag here will be treat as raw string instead of html content, so we need to format back in order to get the tree format